### ISSUE TITLE
Quicktest watch doesn't work anymore

### ISSUE THREAD

**Issue by MangelMaxime** [fable-compiler/Fable#1869]
### Description

@alfonsogarciacaro 

I think this commit https://github.com/fable-compiler/Fable/commit/caddbdea801619a39d73fee880aa69fe051fffa8 brokes the watch support of `npm run build quicktest`.

### Repro code

1. Run `npm run build quicktest`
2. Modify `Quicktest.fs`
3. Nothing happens

### Expected and actual results

It should recompile and re-run the quicktest file.

**Comment by alfonsogarciacaro** [comment]
Hi @MangelMaxime! I replaced chokidar-cli with nodemon, did you run `npm install` before `npm run build quicktest`?

**Comment by MangelMaxime** [comment]
@alfonsogarciacaro I didn't perhaps that's the problem but I wonder why it's not complaining about `nodemon` being not found.

**Comment by MangelMaxime** [comment]
I tried to run `npm install` first but still not working.

`nodemon` did start as before but saving the file doesn't trigger a rebuild.

**Comment by alfonsogarciacaro** [comment]
Strange. I just run the following commands with latest master and it worked :thinking;

```shell
git clean -xfd  # just to be sure
npm i
npm run build quicktest
```

**Comment by MangelMaxime** [comment]
Same result...

If I run `npx nodemon --watch src/quicktest -e "fs" --exec "fable-splitter -c src/quicktest/splitter.config.js --run"` then it works.

**Comment by alfonsogarciacaro** [comment]
Ah! You made me realize I forgot add `npx` before `nodemon`. Probably it only worked in my machine because I must have nodemon installed globally. Anyways, I'm also using your version now because it watches also other .fs files. Hope it works for you too now!

**Comment by MangelMaxime** [comment]
Will test it tomorrow when I have access to my mac.

I also had `nodemon` globally installed and nodemon was starting and saying "watching blabla".

It just didn't pick the changes, and I don't know why.

**Comment by alfonsogarciacaro** [comment]
`quicktest` uses now dotnet watch so hopefully the issue is fixed, please reopen if you still have problems
